evap/results/tests/test_views.py

```diff
@@ -52,12 +52,29 @@ def setUpTestData(cls):
         mommy.make(Contribution, course=cls.course, contributor=responsible, can_edit=True, responsible=True, comment_visibility=Contribution.ALL_COMMENTS)
         mommy.make(Contribution, course=cls.course, contributor=contributor, can_edit=True)
 
+        # Private course
+        student = UserProfile.objects.get(username="student")
+        test1 = mommy.make(UserProfile, username="test1")
+        test2 = mommy.make(UserProfile, username="test2")
+        cls.private_course = mommy.make(Course, state='published', is_private=True, semester=cls.semester, participants=[student, test1, test2], voters=[test1, test2])
+        mommy.make(Contribution, course=cls.private_course, contributor=responsible, can_edit=True, responsible=True, comment_visibility=Contribution.ALL_COMMENTS)
+        mommy.make(Contribution, course=cls.private_course, contributor=contributor, can_edit=True)
```

Comment:
could you move the generation of data for specific tests to those specific tests?